src/index.html

```diff
@@ -1,13 +1,50 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com/%22%3E">
+    <link rel="preconnect" href="https://fonts.gstatic.com/%22%3E">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="container" data-qa="card">
+      <img
+        class="container__logo"
+        src="images/imac.jpeg"
+        alt="tovar_logo"
+      >
+
+      <h2 class="conainer_info">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h2>
+      <p class="conainer_kod">Код товара: 195434</p>
+
+      <div class="container__review">
+        <div class="stars stars--4">
+          <div class="star"></div>
+          <div class="star"></div>
+          <div class="star"></div>
+          <div class="star"></div>
+          <div class="star"></div>
+        </div>
+
+        <p class="container__review-col">Отзывов: 5</p>
+      </div>
+
+      <div class="container__buy">
+        <p class="price">Цена:</p>
+        <p class="price-grn">69 999 грн</p>
```

Comment:
you have 2 differen blocks for price label and actual price, why? just make them elements of your card block

Reply:
pay attention to my comments, you don't need 2 blocks for price, make them elements of card block